optuna/visualization/matplotlib/__init__.py

```diff
@@ -0,0 +1,8 @@
+from optuna.visualization.matplotlib._contour import plot_contour  # NOQA
```

Comment:
I'm not sure, but how about not listing functions that are not yet implemented? When users look at the documentation, they may be confused by the lack of documentation or by the unnatural type of functions.

Reply:
I would like to discuss this point with @ytknzw and other reviewers. Thank you!

Reply:
> I'm not sure, but how about not listing functions that are not yet implemented?

I agree with his suggestion.

Reply:
I agree with you two. I didn't think well that part, sorry.